quantum/quantum.c

```diff
@@ -1011,6 +1011,7 @@ void matrix_scan_quantum() {
   #endif
 
   matrix_scan_kb();
+    
```

Comment:
Could you remove this change? 
```suggestion
```

Reply:
Yes. I will revert this in my next update.